docs/generate_api_reference.py

```diff
@@ -35,6 +77,27 @@ def module_doc_path(module):
     )
 
 
+def module_flowchart_path(module, local=False):
+    """
+    Get the path for a module's mermaid flowchart file.
+
+    If local is True, returns the absolute path to the file.
+    If local is False, returns the path relative to the module's documentation file.
+
+    :param module: Module to get flowchart path for.
+    :param local: Whether to return the absolute path to the file.
```

Comment:
```suggestion
    :param local: Whether to return the absolute path to the file.
        Default is False.
```

Reply:
basic question: why do you need both options?

Reply:
I use the absolute path to check for existence of the .mmd file and the relative path within the .rst files that create the HTMLs. If a package/module doesn't have an mmd file then no flowchart is added to the docs